Lares/Assets/Mechanics/Inventory/Runtime/ConsumableItems.cs

```diff
@@ -0,0 +1,85 @@
+using UnityEngine;
+
+namespace Lares.Inventory
+{
+    public class HealthConsumable : InventoryItem, IConsumable
+    {
+        [SerializeField] private int _percentageIncrease;
+        public override void OnUse(GameObject playerRef)
+        {
+            Debug.Log("Health potion used " + _percentageIncrease);
+        }
+    }
+
+    public class MagicConsumable : InventoryItem, IConsumable
+    {
+        [SerializeField] private int _percentageIncrease;
+        public override void OnUse(GameObject playerRef)
+        {
+            Debug.Log("Magic potion used");
+        }
+    }
+
+    public class PhysicalAttackConsumable : InventoryItem, IConsumable
+    {
+        [SerializeField] private int _PAtkModifier;
```

Comment:
This and all the next `_X` are wrong due to the same reason why `_id` is correct and not `_iD` or `_ID`
```suggestion
        [SerializeField] private int _pAtkModifier;
```
or
```suggestion
        [SerializeField] private int _phyAtkModifier;
```